KomaMRIBase/src/datatypes/Phantom.jl

```diff
@@ -612,5 +539,56 @@ function check_phantom_arguments(nd, ss, us)
             ssy = ss[2]
         end
     end
+    
     return ssx, ssy, ssz, usx, usy, usz
 end
+
+"""
+    ρ, T1, T2, T2s, Δw = default_brain_tissue_properties(labels, tissue_properties = nothing)
+
+This function returns the default brain tissue properties using a labels identifier Matrix
+# Arguments
+- `labels` : (`::Matrix`) the labels identifier matrix of the phantom
+- `tissue_properties` : (`::Dict`, `=Dict()`) phantom tissue properties in ms and Hz considering the available tissues
+
+# Returns
+- `ρ, T1, T2, T2s, Δw`: (`::Matrix`) matrices of the same size of labels with the tissues properties information
+
+# Examples
+```julia-repl
+julia> ρ, T1, T2, T2s, Δw = default_brain_tissue_properties(labels, tissue_properties)
+
+julia> ρ, T1, T2, T2s, Δw = default_brain_tissue_properties(labels)
+```
+"""
+function default_brain_tissue_properties(labels, tissue_properties = Dict())
+    # Load default tissue properties
+    default_properties = Dict(
+        # ρ, T1, T2, T2*, Δw
+        "CSF"           => [1,      2.569,  0.329,  0.058,  0],
+        "GM"            => [0.86,   0.833,  0.083,  0.069,  0],
+        "WM"            => [0.77,   0.500,  0.070,  0.061,  0],
+        "FAT1"          => [1,      0.350,  0.070,  0.058,  -220*2π], #-220 Hz
+        "MUSCLE"        => [1,      0.900,  0.047,  0.030,  0],
+        "SKIN/MUSCLE"   => [1,      0.569,  0.329,  0.058,  0],
+        "SKULL"         => [0,      0,      0,      0,      0],
+        "VESSELS"       => [0,      0,      0,      0,      0],
+        "FAT2"          => [0.77,   0.500,  0.070,  0.061,  -220*2π], #-220 Hz
+        "DURA"          => [1,      2.569,  0.329,  0.058,  0],
+        "MARROW"        => [0.77,   0.500,  0.070,  0.061,  0])
+    
+    tissue_properties = merge(default_properties, tissue_properties)
+    props = ["ρ", "T1", "T2", "T2s", "Δw"]
+    Nproperties = length(props)
+    # Order: CSF, DURA, FAT1, FAT2, GM, MARROW, MUSCLE, SKIN/MUSCLE, SKULL, vESSELS, WM
+    tissue_labels = [23, 232, 93, 209, 46, 255, 116, 139, 162, 185, 70]
```

Comment:
Could you try one last thing? 

The sorting and use of `tissue_texts`, does not make a lot of sense.  It is better if `tissue_labels` is also a dictionary (`Dict("CSF" => 23, ...)`), and we index it by the `default_properties` keys.

You should be able to do:

```julia
for i=1:Nproperties
    for (tissue, property) in tissue_properties
        data_properties[i, :, :, :] += (labels .== tissue_labels[tissue]) * property[i]
    end
end
```
Also, don't forget the spaces between the `*`!